uui-v/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@epam/uui-v",
+  "version": "4.5.3",
+  "description": "EPAM UUI-V components set",
+  "author": "EPAM",
+  "license": "MIT",
+  "main": "index.js",
```

Comment:
add   `"private": true,`, because we don't plan to publish this package to npm now